docs/advanced-topics/ideal_integration.md

```diff
@@ -59,111 +65,106 @@ Examples:
+flags, the build command for `foo_fuzzer` looks similar to this:
 
 ```bash
 # Assume the following env vars are set:
 # CC, CXX, CFLAGS, CXXFLAGS, LIB_FUZZING_ENGINE
 $ make_or_whatever_other_command foo_fuzzer
 ```
 
-This will allow to have minimal OSS-Fuzz-specific configuration and thus be
-more robust.
+This minimizes OSS-Fuzz-specific configuration, making your fuzzing more robust.
 
 There is no point in hardcoding the exact compiler flags in the build system
-because they a) may change and b) are different depending on the fuzzing engine
-and the sanitizer being used.
+because they a) may change and b) depend on the fuzzing engine and sanitizer
+being used.
 
 ## Seed Corpus
-The *corpus* is a set of inputs for the fuzz target (stored as individual files). 
-When starting the fuzzing process, one should have a "seed corpus", 
-i.e. a set of inputs to "seed" the mutations.
-The quality of the seed corpus has a huge impact on fuzzing efficiency as it
-allows the fuzzer to discover new code paths more easily.
 
-The ideal corpus is a minimal set of inputs that provides maximal code coverage. 
+The *seed corpus* is a set of test inputs, stored as individual files, provided
+to the fuzz target as a starting point (to "seed" the mutations). The quality of
+the seed corpus has a huge impact on fuzzing efficiency; the higher the quality,
+the easier it is for the fuzzer to discover new code paths. The ideal corpus is
+a minimal set of inputs that provides maximal code coverage. 
 
-For better OSS-Fuzz integration, 
-the seed corpus should be available in revision control (can be same or
-different as the source code). It should be regularly extended with the inputs
-that (used to) trigger bugs and/or touch new parts of the code. 
+For better OSS-Fuzz integration,  the seed corpus should be available in
+revision control (it can be the same as or different from the source code). It
+should be regularly extended with the inputs that (used to) trigger bugs and/or
+touch new parts of the code. 
 
 Examples: 
 [boringssl](https://github.com/google/boringssl/tree/master/fuzz),
 [openssl](https://github.com/openssl/openssl/tree/master/fuzz),
 [nss](https://github.com/mozilla/nss-fuzzing-corpus) (corpus in a separate repo).
 
 ## Dictionary
+
 For some input types, a simple dictionary of tokens used by the input language
-can have a dramatic positive effect on fuzzing efficiency. 
-For example, when fuzzing an XML parser, a dictionary of XML tokens will help.
-AFL has a [collection](https://github.com/rc0r/afl-fuzz/tree/master/dictionaries)
-of such dictionaries for some of the popular data formats.
-Ideally, a dictionary should be maintained alongside the fuzz target.
-The syntax is described [here](http://libfuzzer.info/#dictionaries).
+can have a dramatic impact on fuzzing efficiency.  For example, when fuzzing an
+XML parser, a dictionary of XML tokens is helpful. AFL has a
+[collection](https://github.com/rc0r/afl-fuzz/tree/master/dictionaries) of
```

Comment:
Kevin, can you please replace this link with https://github.com/google/AFL/tree/master/dictionaries ?

Reply:
Done